controller/src/main/java/io/pravega/controller/server/rpc/auth/PravegaAuthManager.java

```diff
@@ -57,28 +59,77 @@ private AuthHandler getHandler(String handlerName) throws AuthenticationExceptio
      *                 Returns false if the entity does not have access. 
      * @throws AuthenticationException if an authentication failure occurred.
      */
-    public boolean authenticate(String resource, String credentials, AuthHandler.Permissions level) throws AuthenticationException {
+    public boolean authenticateAuthorize(String resource, String credentials, AuthHandler.Permissions level) throws AuthenticationException {
```

Comment:
Style comment: it should be `authenticateAndAuthorize`, no?

Reply:
Fixed.